--- a/src/PantheonTerminalWindow.vala
+++ b/src/PantheonTerminalWindow.vala
@@ -161,6 +161,7 @@
             notebook.tab_duplicated.connect (on_tab_duplicated);
             notebook.allow_new_window = true;
             notebook.allow_duplication = true;
+            notebook.margin_top = 3;
             notebook.group_name = "pantheon-terminal";
 
             notebook.tab_added.connect ((tab) => {
